src/open_inwoner/configurations/models.py

```diff
@@ -105,25 +105,25 @@ class SiteConfiguration(SingletonModel):
     )
     home_theme_title = models.CharField(
```

Comment:
This series of fields are all still named with 'theme'.

We could also change them back to 'category' in this PR, or create a new ticket because it is some work (I know these are copied into the template-context and used in templates).

Reply:
We discussed this and we're not going to rename the translation fields, because we're likely moving to django-cms which has translations built in.